apps/browser/src/vault/popup/components/vault/add-edit.component.ts

```diff
@@ -128,6 +128,14 @@ export class AddEditComponent extends BaseAddEditComponent {
       await this.load();
 
       if (!this.editMode || this.cloneMode) {
+        // Only allow setting username if there's no existing value
+        if (
+          params.username &&
+          (this.cipher.login.username === null || this.cipher.login.username === "")
```

Comment:
```suggestion
          (this.cipher.login.username == null || this.cipher.login.username === "")
```

Kind of a nit but also kind of not because JavaScript is a great language lol.